Don't depend on devel/git as distfiles are generated by git-archive(1)
thus don't contain .git subdirs.

--- externals/nx_tzdb/CMakeLists.txt.orig	2023-06-23 13:27:28 UTC
+++ externals/nx_tzdb/CMakeLists.txt
@@ -5,15 +5,11 @@ add_library(nx_tzdb INTERFACE)
 
 add_library(nx_tzdb INTERFACE)
 
-find_program(GIT git)
 find_program(GNU_MAKE make)
 find_program(DATE_PROG date)
 
 set(CAN_BUILD_NX_TZDB true)
 
-if (NOT GIT)
-    set(CAN_BUILD_NX_TZDB false)
-endif()
 if (NOT GNU_MAKE)
     set(CAN_BUILD_NX_TZDB false)
 endif()
--- externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/version.orig	2023-06-23 13:27:28 UTC
+++ externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/version
@@ -0,0 +1 @@
+2022c
--- externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt.orig	2023-06-23 19:37:14 UTC
+++ externals/nx_tzdb/tzdb_to_nx/src/tzdb/CMakeLists.txt
@@ -1,8 +1,3 @@
-find_program(GIT_PROGRAM git)
-if ("${GIT_PROGRAM}" STREQUAL "GIT_PROGRAM-NOTFOUND")
-    message(FATAL_ERROR "git program not found")
-endif()
-
 find_program(GNU_DATE date)
 if ("${GNU_DATE}" STREQUAL "GNU_DATE-NOTFOUND")
     message(FATAL_ERROR "date program not found")
@@ -12,17 +7,7 @@ set(TZDB_VERSION_FILE ${TZ_SOURCE_DIR}/NEWS)
 set(NX_ZONEINFO_DIR "${NX_TZDB_DIR}/zoneinfo")
 
 set(TZDB_VERSION_FILE ${TZ_SOURCE_DIR}/NEWS)
-
-execute_process(
-    COMMAND 
-        ${GIT_PROGRAM} log --pretty=%at -n1 NEWS
-    OUTPUT_VARIABLE
-        TZ_COMMIT_TIME
-    WORKING_DIRECTORY
-        ${TZ_SOURCE_DIR}
-    COMMAND_ERROR_IS_FATAL ANY)
-
-string(REPLACE "\n" "" TZ_COMMIT_TIME "${TZ_COMMIT_TIME}")
+set(TZ_COMMIT_TIME 1687220316)
 
 if (APPLE)
 	set(VERSION_COMMAND ${GNU_DATE} -r ${TZ_COMMIT_TIME} +%y%m%d) 
